src/main/java/org/broadinstitute/hellbender/utils/codecs/gencode/GencodeGtfCodec.java

```diff
@@ -139,16 +139,9 @@ public FeatureCodecHeader readHeader(final LineIterator lineIterator) throws IOE
         return new FeatureCodecHeader(readActualHeader(lineIterator), FeatureCodecHeader.NO_HEADER_END);
     }
 
-    @SuppressWarnings( "deprecation" )
     @Override
     public LocationAware makeIndexableSourceFromStream(final InputStream bufferedInputStream) {
-        final PositionalBufferedStream pbs;
-        if (bufferedInputStream instanceof PositionalBufferedStream) {
-            pbs = (PositionalBufferedStream) bufferedInputStream;
-        } else {
-            pbs = new PositionalBufferedStream(bufferedInputStream);
-        }
-        return new AsciiLineReaderIterator(new AsciiLineReader(pbs));
+        return new AsciiLineReaderIterator(AsciiLineReader.from(bufferedInputStream));
```

Comment:
Is this method guaranteed to wrap the stream in a `PositionalBufferedStream`?

Reply:
Yes